plugins/inputs/deprecations.go

```diff
@@ -20,6 +20,10 @@ var Deprecations = map[string]telegraf.DeprecationInfo{
 		Since:  "1.6.0",
 		Notice: "use 'inputs.http' instead",
 	},
+	"icinga2": {
+		Since:  "1.26.0",
+		Notice: "has removed icinga2.object_type (string), use icinga2.objects = [] (array) instead",
+	},
```

Comment:
This file is only for deprecated plugins themselves, not attributes, so you can get rid of this.

Reply:
Thanks, I've already removed it.